ddsc/core/ddsapi.py

```diff
@@ -448,7 +477,7 @@ def get_users_by_full_name(self, full_name):
         data = {
             "full_name_contains": full_name,
         }
-        return self._get('/users', data, content_type=ContentType.form)
+        return self._get_all_pages('/users', data, content_type=ContentType.form)
 
     def get_users_by_page_and_offset(self, page, per_page):
```

Comment:
Looks like there was already a users listing method that paginated? Is this still necessary?